crates/swc_ecma_utils/src/factory.rs

```diff
@@ -60,6 +60,18 @@ pub trait ExprFactory: Into<Expr> {
         })
     }
 
+    #[inline]
+    fn call_fn(self, args: Vec<ExprOrSpread>, span: Span) -> Expr {
```

Comment:
Can you change the order of arguments?
I think span should come first for consistency.